<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>winner_4nafare_game</class>
 <widget class="QMainWindow" name="winner_4nafare_game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>711</width>
      <height>511</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border : 4px solid #10ac84 ;
background-color : white ;</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="oneperson_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>180</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid gold ;
text-align : center ;
font: 20pt &quot;Bodoni MT Condensed&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>151</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/gold.png);
border : none ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>110</y>
       <width>151</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/copper.png);
border : none ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="threeperson_label">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>270</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px outset #c23616 ;
text-align : center ;
font: 20pt &quot;Bodoni MT Condensed&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="fourperson_label">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>390</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px solid black ;
text-align : center ;
font: 20pt &quot;Bodoni MT Condensed&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="twoperson_label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>270</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border : 2px inset grey ;
text-align : center ;
font: 20pt &quot;Bodoni MT Condensed&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>171</width>
       <height>141</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/silver.png);
border : none ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
